src/backend/storeManagers/gog/library.ts

```diff
@@ -948,6 +948,14 @@ export async function gogToUnifiedInfo(
     ?.replace('{formatter}', '')
     .replace('{ext}', 'jpg')
 
+  const icon =
+    info.game?.square_icon.url_format
+      ?.replace('{formatter}', '')
+      .replace('{ext}', 'jpg') ||
+    info.game?.icon?.url_format
+      ?.replace('{formatter}', '')
+      .replace('{ext}', 'jpg')
```

Comment:
Nitpicking, but would
```suggestion
  const icon = (info.game.square_icon.url_format || info.game.icon?.url_format)
    ?.replace('{formatter}', '')
    .replace('{ext}', 'jpg')
```
work?

Reply:
good point